pkg/api/v1/types.go

```diff
@@ -3291,6 +3291,8 @@ const (
 	NodeNetworkUnavailable NodeConditionType = "NetworkUnavailable"
 	// NodeInodePressure means the kubelet is under pressure due to insufficient available inodes.
 	NodeInodePressure NodeConditionType = "InodePressure"
+	// NodeKernelDeadlock means the kernel of a node has a temporary or permanent deadlock.
```

Comment:
This should be clearly documented that `NodeKernelDeadlock` is updated from node-problem-detector. In order to get this condition, users should deploy a node-problem-detector. :)